docs/2_Easy_Fix_Difficult_Test.md

```diff
@@ -190,7 +190,7 @@ in the code and provided us with quick fixes!
   +               const auto& invoice = *id2Invoice.second;
   +               totalAmount += finance::toUSD(invoice.computeTotalAmount(), invoice.getCountry().getCurrency());
           }
-          return totalAmount;
+          return ReportGenerator::getRoundedValueOf(totalAmount);
```

Comment:
`getRoundedValueOf `could be a free function rather than a class static member function. Especially since it does not belong to the abstraction layer of `ReportGenerator`